topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorBaseIT.java

```diff
@@ -200,7 +200,7 @@ public void teardown() throws InterruptedException, TimeoutException, ExecutionE
                 // Wait for the operator to delete all the existing topics in Kafka
                 for (KafkaTopic item : items) {
                     LOGGER.info("Deleting {} from Kube", item.getMetadata().getName());
-                    operation().inNamespace(NAMESPACE).withName(item.getMetadata().getName()).cascading(true).delete();
+                    operation().inNamespace(NAMESPACE).withName(item.getMetadata().getName()).delete();
```

Comment:
I think there's another place in `TopicOperatorBaseIT` where we do a cascading delete (see [85eaa7fe](https://github.com/strimzi/strimzi-kafka-operator/commit/85eaa7fea4fee32ea0582dd918a4e8f1fe3d87da#diff-25da95bbedc25f0bd60e013de43219c7R600))

Reply:
I reverted just https://github.com/strimzi/strimzi-kafka-operator/pull/2207/files for now... If the test results will be ok, I will search for other occurrences. However I did not know there any other, so thank you for pointing it out.